core/upgrade/upgrades.php

```diff
@@ -857,6 +857,10 @@ public static function _v_3_17_0_site_settings_updates() {
 		}
 	}
 
+	public static function _v_3_21_0_move_gutenberg_loading_to_settings() {
+		add_option( 'optimized_gutenberg_loading', '1' );
```

Comment:
Why should you add this option to the DB?

Reply:
@KingYes Because all the other options are also stored in the DB.